samples/sensor/accel_trig/boards/nrf52dk_nrf52832.overlay

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2025 Pierrick Curt
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <zephyr/dt-bindings/sensor/lis2dw12.h>
+
+/ {
+	aliases {
+		accel0 = &lis2dw12_19_x_nucleo_iks01a3;
+	};
+};
+
+&arduino_i2c {
+
+	lis2dw12_19_x_nucleo_iks01a3: lis2dw12@19 {
+		status = "okay";
+		compatible = "st,lis2dw12";
+		reg = <0x19>;
+		odr = <400>;
+		tap-mode = <LIS2DW12_DT_SINGLE_DOUBLE_TAP>;
+		tap-threshold = <12>, <12>, <12>;
+		power-mode = <LIS2DW12_DT_HP_MODE>;
+		tap-shock = <0x03>;
+		tap-quiet = <0x03>;
+		tap-latency = <0x03>;
+		irq-gpios =  <&arduino_header 3 GPIO_ACTIVE_HIGH>; /* A3 */
+	};
```

Comment:
This can be simplified to
```suggestion
&lis2dw12_19_x_nucleo_iks01a3 {
	status = "okay";
	odr = <400>;
	tap-mode = <LIS2DW12_DT_SINGLE_DOUBLE_TAP>;
	tap-threshold = <12>, <12>, <12>;
	power-mode = <LIS2DW12_DT_HP_MODE>;
	tap-shock = <0x03>;
	tap-quiet = <0x03>;
	tap-latency = <0x03>;
};
```
And with some cmake magic, you could move it to a common overlay that gets conditionally applied when SHIELD=x_nucleo_iks01a3, instead of duplicating it for nrf52dk_nrf52832 and stm32f3_disco.

Reply:
Yes, thanks for the suggestion